README.md

```diff
@@ -72,7 +72,7 @@ foreach (var stat in varusRanked.Stats)
 }
 ```
 
-You can find a list of all the available operations in [IRiotApi](RiotSharp/IRiotApi.cs).
+You can find a list of all the available operations in [TournamentApi in Documentation](http://benfradet.github.io/RiotSharp/api/RiotSharp.RiotApi.html).
```

Comment:
I think we should still mention the class holding the API being discussed if someone wants to dig into the code without looking at the doc.